recipes/percolator/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: percolator
+  version: '3.0'
+
+source:
+  fn: rel-3-00.tar.gz
+  url: https://github.com/percolator/percolator/archive/rel-3-00.tar.gz
+
+build:
+  number: 0
+  string: boost{{CONDA_BOOST}}_{{PKG_BUILDNUM}}
```

Comment:
You probably want to hardcoed the conda_boost here. Currently the name looks like this `/anaconda/conda-bld/linux-64/percolator-3.0-boost1.60_0.tar.bz2, `